### DISCUSSION TITLE
Nested Dialog Focus Problem

### DISCUSSION THREAD

**Discussion by sghong3130** [tailwindlabs/headlessui#2983] [General]
When the modals are nested like elements below, second modal (z-index 102) can't be focused until it is clicked.
Any solutions?

<img width="522" alt="스크린샷 2024-02-13 오후 1 31 01" src="https://github.com/tailwindlabs/headlessui/assets/124551834/4cf55603-010c-4feb-a2b2-b5f14fb4689f">

**Comment by sghong3130** [comment]
I found this issue was fixed on "@headlessui/react":"2.0.0-alpha.4" under. Thanks.

**Reply by roder125** [reply]
What about the vue package of headless ui? I get the same problem with the latest release of version [1.7.19](https://www.npmjs.com/package/@headlessui/vue/v/1.7.19)

**Reply by sghong3130** [reply]
https://github.com/tailwindlabs/headlessui/releases/tag/%40headlessui%2Freact%40v2.0.0-alpha.1

I think this release fixed this issue, but it seems that there are no similar releases in vue package
